libs/martech/attributes.js

```diff
@@ -82,8 +82,9 @@ export function decorateDefaultLinkAnalytics(block, config) {
 }
 
 export async function decorateSectionAnalytics(section, idx, config) {
+  const id = Number.isNaN(idx) ? `s${idx + 1}` : idx;
```

Comment:
sorry, I think I'm seeing issues with !Number.isNaN in the modal tests. maybe try this:
```suggestion
  const id = Number.isInteger(idx) ? `s${idx + 1}` : idx;
```